CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ cd material-ui
 git remote add upstream git@github.com:mui-org/material-ui.git
 ```
 
-3. **The `master` branch represents the development as well as the release branch for the latest version (v4) of Material-UI.** The other branches are for legacy versions or upcoming changes of Material-UI. Any work you do should be based on the latest upstream `master` branch. So before starting to work on a new topic, always synchronize your local `master` branch with the upstream one:
+3. **The `master` branch represents the development as well as the release branch for the latest version (v4) of Material-UI.** The other two branches (`v0.x` and `v3.x`) are for legacy versions of Material-UI. Any work you do should be based on the latest upstream `master` branch. So before starting to work on a new topic, always synchronize your local `master` branch with the upstream one:
```

Comment:
Could you rewrite so that it doesn't specifically say that there are "two other". Just say what 0.x and 3.x are for. Anything locking the number will cause confusion if it gets outdated.

Reply:
I heard from @oliviertassinari that we will introduce "next" branch for upcoming work, which will basically change the entire branch structure we use today. So this "branches" section will have to be completly refactored anyways very soon. I disagree with @oliviertassinari about how the branch structure itself should look in the future. 

There will probably have to be a subsection "branch structure" to explain all specialties if we take olivier's approach. 

I suggest we take "lazy" approach and leave it as is (with "two"), and adjust when the branch structure change, if you agree?

Reply:
> I suggest we take "lazy" approach and leave it as is, and adjust when the branch structure changes, if you agree?

Well as I said I don't. These documents are rarely updated and there's no need to lock it to there are "two others". Just say what 0.x and 3.x are for and no information is lost while also causing no confusion if someone accidentally creates a branch in the upstream.

Reply:
Yes. I missunderstood your comment a bit. I'll do that in a second.

Reply:
You're focusing way too much on the branch "structure". This is something you almost never are concerned with. 

99.999% go into master. If not then a maintainer will correct it. If we have to repeat ourselves very often about a specific topic then can we add a section about it. Otherwise "target your PR at master" is straight forward and covers almost all cases.